### PR TITLE
RA-1722: General actions is an empty list when editing patient info

### PR DESCRIPTION
**Pull request by gitcliff**
>General actions is an empty list when editing patient info

https://issues.openmrs.org/browse/RA-1722

### CODE REVIEW

**Comment by sherrif10** [comment]
@gitcliff  is it possible to attach the screen shot thanks

**Comment by gitcliff** [comment]
@sherrif10 i attached the screen shots

**Comment by mozzy11** [comment]
@mks-d  @dkayiwa   ,LGTM

File: omod/src/main/webapp/fragments/summary/registrationSummary.gsp [hunk 1]
```diff
@@ -40,21 +40,4 @@
...
-                            <a href="${ ui.encodeJavaScript(ext.url("/" + ui.contextPath(), appContextModel, returnUrl)) }" id="${ ext.id }">
-                                <li>
-                                    <i class="${ ext.icon }"></i>
-                                    ${ ui.message(ext.label) }
-                                </li>
-                            </a>
-                    <% } %>
-                </ul>
-            </div>
-        </div>
```
Comment (mks-d) [inline omod/src/main/webapp/fragments/summary/registrationSummary.gsp]: What I said [here](https://issues.openmrs.org/browse/RA-1722?focusedCommentId=262488&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-262488) is that it is ok to not see the _same_ 'General Actions' than those on the clinical dashboard. But I didn't say that the Registration Summary would not allow for the configuration of its own 'General Actions'.

Those `overallActions`, where/how are they fetched?

Comment (gitcliff) [reply omod/src/main/webapp/fragments/summary/registrationSummary.gsp]: @mks-d am sorry sbout the delay in the replying to this ,unfortunately just seen this ,
the `overallActions `are enabled extensions for the current user from a specified` extensionPointId` and they a bundled to the model from the` RegistrationSummaryFragmentController`

**Comment by brandones** [comment]
Closing due to comments in https://issues.openmrs.org/browse/RA-1722 saying this is already fixed.

**Pull request closed without merging**